lib/game_functions.rb

```diff
@@ -28,9 +28,11 @@ def player_turn(player_input, piece)
 
     def computer_turn
         computer_choice = @counters.keys.sample
+        while !valid_column?(computer_choice)
+            computer_choice = @counters.keys.sample
+        end
```

Comment:
I'm not sure if this was the best fix, but I found a bug when playing the game that was still allowing the computer to choose an invalid column